ovis_bringup/launch/ovis_arm.launch

```diff
@@ -0,0 +1,27 @@
+<launch>
+
+  <arg name="use_urdf" default="false" />
```

Comment:
Why not use_urdf like in ovis_bringup/launch/kinova_driver.launch? More precisely, why this launch file setup vs the other one?